src/CoreVideo/CVBuffer.cs

```diff
@@ -150,9 +151,9 @@ public NSObject GetAttachment (NSString key, out CVAttachmentMode attachmentMode
 			if (key == null)
 				throw new ArgumentNullException ("key");
 			if (PlatformHelper.CheckSystemVersion (12, 0))
-				return Runtime.GetNSObject (CVBufferCopyAttachment (handle, key.Handle, out attachmentMode));
+				return Runtime.GetINativeObject<NSObject> (CVBufferCopyAttachment (handle, key.Handle, out attachmentMode), true);
```

Comment:
`Runtime.GetNSObject<NSObject> (ptr, true)`

since `GetINativeObject` logic is more complex and you already know the return type.